openedx/core/lib/new_relic_utils.py

```diff
@@ -0,0 +1,24 @@
+"""
+Contains utilities for working with New Relic.
```

Comment:
Let's discuss.  I don't know the right answer.  However, I know that when I was doing our platform inventory project I found it annoying to keep finding generically named files including a single non-generic method.  

The question is when we don't have a coherent library of utilities, and we just have a single utility method, what is the best way to have it found, understood, and potentially grow into a library.  I was leaning toward more specific to start and grouping into more generic files (or directories) as more gets added.  

But, unfortunately, copy/paste is the most likely way for this utility to get re-used, and not due to searching for it or seeing appropriate names.  

Anyway, it's an open question I have about how to get the most reuse from any utilities.  Ultimately we'll need searchable documentation, especially as some of these get moved to a separate repo.